portal/views/portal.py

```diff
@@ -595,10 +595,34 @@ def home():
 def admin():
     """user admin view function"""
     # can't do list comprehension in template - prepopulate a 'rolelist'
-    users = User.query.filter_by(deleted=None).all()
+
+    request_org_list = request.args.get('org_list', None)
+
+    if request_org_list:
+        org_list = set()
+
```

Comment:
Well, we're doing something _similar_ in the /patients page. But that logic is different enough (because there, `org_list` also depends on the `current_user`'s orgs, the relative role levels, etc), that I don't think it's worth trying to break it into a shared method.